sdk/communication/azure-communication-callingserver/azure/communication/callingserver/_callingserver_client.py

```diff
@@ -209,3 +219,42 @@ def join_call(
         )
 
         return CallConnection(join_call_response.call_connection_id, self._call_connection_client)
+
+    @distributed_trace()
+    def start_download(
+        self,
+        content_url,  # type: str
+        **kwargs  # type: Any
+    ):  # type: (...) -> HttpResponse
+        """Start download using content url.
+
+        :param str content_url:
+            The content url.
+        :returns: HttpResponse for a successful download request.
+        :rtype: ~HttpResponse
+        """
+        if not content_url:
+            raise ValueError("content_url can not be None")
+
+        if not CallingServerUtils.is_valid_url(content_url):
+            raise ValueError("content_url is invalid")
+
+        client_models = {k: v for k,
+                         v in _models.__dict__.items() if isinstance(v, type)}
+
+        self._serialize = Serializer(client_models)
+        self._serialize.client_side_validation = False
+        self._deserialize = Deserializer(client_models)
+        self._config = AzureCommunicationCallingServerServiceConfiguration(
+            self._endpoint, authentication_policy=self._authentication_policy)
```

Comment:
Why to create all these variables if we could reuse whatever exists within the `self._calling_server_client`?